internal/pkg/convert/convert.go

```diff
@@ -303,11 +314,23 @@ func getN9Indicator(i v1.SLISpec, project string) nobl9v1alpha.Indicator {
 		MetricSource: nobl9v1alpha.MetricSourceSpec{
 			Project: metricSourceProject,
 			Name:    "Changeme",
-			Kind:    "Agent",
+			Kind:    getKindFromAnnotations(metadata),
 		},
 	}
 }
 
+func getKindFromAnnotations(metadata v1.Metadata) string {
+	if value, ok := metadata.Annotations[n9KindAnnotation]; ok {
+		switch strings.ToLower(value) {
+		case "direct":
+			return kindDirect
+		case "agent":
+			return kindAgent
+		}
+	}
+	return kindAgent
```

Comment:
do you believe user should be warned here? or do you believe relevant docs would be enough?